spec/system/admin_order_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'AdminReward crud', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:admin_user) { create(:admin_user) }
+  let!(:employee) { create(:employee) }
+  let!(:reward) { create(:reward, price: 1) }
+  let(:reward2) { create(:reward, price: 2) }
```

Comment:
do we really need to create the whole object, if you are using only it's price as an input value placeholder?
If you want to keep that value in the let for the sake of visibility consider something like:
```suggestion
  let(:new_price) { 2 }
```